labsc-notebooks/ERA5_series_moy_ann_sais_mens.ipynb

```diff
@@ -0,0 +1,3595 @@
+{
```

Comment:
<p>It would be fairly easy to extract all points into a dataset and assign coordinate of the <code>name</code>   ... would that be more interesting?  </p><p>For the <code>debut</code>,<code>fin</code> I would tend to calculate over the longest then customize in the plotting  ...  </p><p> </p><p><code>plt.plot(ser_ann.sel(time=slice(debut,fin)).time, ser_ann.sel(time=slice(debut,fin)))</code>  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/LabSc/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>